appengine/flexible/api_gateway/gateway/api_gateway.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import sys
+from flask import Flask
+app = Flask(__name__)
+
+#development ports {static: 8003, flask1: 8001, flask2: 8002}
+@app.route('/test')
+def test():
+    return "GATEWAY OPERATIONAL"
+
+@app.route('/')
+def root():
+    res = requests.get('https://static-dot-flask-algo.appspot.com/')
+    return res.content
+
+@app.route('/hello/<service>')
+def say_hello(service):
+    services = {
+        'flask1': { 'url': 'https://flask1-dot-flask-algo.appspot.com/', 'send': False },
```

Comment:
@jonparrott Already done.  I constructed the urls with `GAE_LONG_APP_ID`.  Made your other changes as well and ready for review.